integration_test/src/main.rs

```diff
@@ -258,8 +257,8 @@ fn test_get_block_header_get_block_header_info(cl: &Client) {
     let tip = cl.get_best_block_hash().unwrap();
     let header = cl.get_block_header(&tip).unwrap();
     let info = cl.get_block_header_info(&tip).unwrap();
-    assert_eq!(header.bitcoin_hash(), info.hash);
-    assert_eq!(header.version, info.version);
+    assert_eq!(header.block_hash(), info.hash);
+    assert_eq!(header.version as u32, info.version);
```

Comment:
maybe better to change info.version to i32?

Reply:
Yeah I agree with this. Bumping bitcoin is a major semver bump here anyway too, so it doesn't matter if we have to change fields.

Reply:
Sounds good - changed at https://github.com/rust-bitcoin/rust-bitcoincore-rpc/pull/141/commits/a24d6089b17a36536af2048da6801f9d9a006d1b.